mathics/builtin/inout.py

```diff
@@ -65,6 +66,20 @@
 SymbolSubscriptBox = Symbol("SubscriptBox")
 
 
+class ShowSteps(Predefined):
+    """
+    If this variable is set to `True`, the steps in the evaluation are shown.
+    """
+
+    name = "ShowSteps"
+    attributes = {
+        "Unprotected",
```

Comment:
@mmatera, instead of letting it unprotected, is better to create a rule for setting this True or False (this is how $TraceBuiltins work).